scalameta/parsersJSFacade/src/main/scala/scala/meta/parsersjsfacade/Api.scala

```diff
@@ -0,0 +1,59 @@
+package scala.meta
+package parsersjsfacade
+
+import scala.scalajs.js
+import js.JSConverters._
+import js.annotation._
+
+import parsers._
+
+object ScalaParser {
```

Comment:
Could you explain the difference between the module kinds? What would it mean if we change the parsersJS module to be common-js?